packages/combo-box/src/vaadin-combo-box-light.js

```diff
@@ -118,10 +118,15 @@ class ComboBoxLight extends ComboBoxDataProviderMixin(ComboBoxMixin(ValidateMixi
   }
 
   /**
-   * @return {string}
+   * Override this getter from `InputMixin` to allow using
```

Comment:
**note:** I'm not sure if it is best to start the comment with `override` when explaining the reason we have overridden this or that method. When looking at the JSDoc, it can give the impression that it tells you to override this getter if you want to customize the property name.

Reply:
Yes, it can be misleading in some cases indeed. 
Maybe it should be changed to `Overriding` instead.
Somehow I was using it consistently since Vaadin 22 🤷‍♂️